engineapi/snapshot.go

```diff
@@ -15,13 +15,17 @@ import (
 
 // SnapshotCreate calls engine binary
 // TODO: Deprecated, replaced by gRPC proxy
-func (e *EngineBinary) SnapshotCreate(engine *longhorn.Engine, name string, labels map[string]string) (string, error) {
+func (e *EngineBinary) SnapshotCreate(engine *longhorn.Engine, name string, labels map[string]string,
+	freezeFilesystem bool) (string, error) {
 	args := []string{"snapshot", "create"}
 	for k, v := range labels {
 		args = append(args, "--label", k+"="+v)
 	}
 	args = append(args, name)
 
+	// Ignore freezeFilesystem argument. v1.7.0 engines understand the corresponding freeze-fs CLI flag, but older
+	// engines do not.
```

Comment:
Are we expecting that newer engine will not be called by this CLI command?

Reply:
I think the answer to your question is generally yes, we do not expect it. Of course, it is still possible if you are running a very old instance-manager with the latest engine. The EngineBinary interface was deprecated in `v1.3.0` and is getting more difficult to test with (must run a `v1.3.0` instance-manager with a `v1.7.0` engine). IMO, we should not attempt to support newer CLI API calls with this interface unless they are critical to stability. WDYT?

Reply:
> IMO, we should not attempt to support newer CLI API calls with this interface unless they are critical to stability
I agree!